<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectProperty</class>
 <widget class="QDialog" name="ProjectProperty">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project property</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>471</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="toolTip">
           <string>The full name of the module</string>
          </property>
          <property name="text">
           <string>Module name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditBibleName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>The short name of the module (reduction).</string>
          </property>
          <property name="text">
           <string>Short module name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditBibleShortName">
          <property name="maxLength">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="toolTip">
           <string>Copyright</string>
          </property>
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditCopyright"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Module Type (Bible or comments or a book).&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Y &lt;/span&gt;if it is put to the Bible or commentary, &lt;span style=&quot; font-weight:600;&quot;&gt;N&lt;/span&gt; if it is a book.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Type Module:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxVersion">
          <property name="maximum">
           <double>150.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="cbmoduleType">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Bible</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabMore">
      <attribute name="title">
       <string>More</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>In module:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cbOldTestament">
           <property name="text">
            <string>Old Testament</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cbNewTestament">
           <property name="text">
            <string>New Testament</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cbApocrypha">
           <property name="text">
            <string>Apocrypha</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="cbChapterZero">
           <property name="text">
            <string>Chapter zero</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="cbStrongNumber">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Strong numbers</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lEStrongDir"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonStrongDir">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBoxSound">
             <property name="text">
              <string>Sound</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lESoundDir"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSoundDir">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="cbNoForcedLineBreaks">
           <property name="text">
            <string>Noforced line breaks</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QSplitter" name="splitter_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QCheckBox" name="cbUseChapterHead">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Use chapter head</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbUseRightAlignment">
             <property name="text">
              <string>Use right alignment</string>
             </property>
            </widget>
           </widget>
           <widget class="QCheckBox" name="cbEnglishPsalms">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>English Psalms</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSplitter" name="splitter_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Categories:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lECategories"/>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSplitter" name="splitter_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Language:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comBLanguage">
            <item>
             <property name="text">
              <string>rus</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>eng</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ukr</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Encoding:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comBEncoding">
            <item>
             <property name="text">
              <string>utf-8</string>
             </property>
            </item>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>471</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Install fonts:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lEInstallFonts"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Desired font name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lEDesiredFontName"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Desired font path:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lEDesiredFontPath"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Desired UI font:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lEDesiredUIFont"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
